src/components/form/FCheckbox.vue

```diff
@@ -101,7 +104,12 @@ FField.FCheckbox(
   use-font('caption')
   user-select none
   margin-left rem(8)
+  margin-top rem(3)
   cursor pointer
+  max-width rem(640)
+
+.FCheckbox__labelText a
+  use-font('caption')
```

Comment:
I think not only `a` we want to be the size of `caption` but all the elements inside, what do you think ?
```suggestion
.FCheckbox__labelText *
  use-font('caption')
```

Reply:
Or maybe even cleaner, remove `use-font('caption')` from `.FCheckbox__label` and do this specificity:
```suggestion
.FCheckbox__label .FCheckbox__labelText
  use-font('caption')
```